src/main/osd/osd_elements.c

```diff
@@ -992,6 +993,21 @@ static void osdElementFlymode(osdElementParms_t *element)
     }
 }
 
+static void osdElementReadyMode(osdElementParms_t *element)
+{
```

Comment:
Move the `static bool isFlying = false;` to here as it's only used in this routine.